test/eventHandlers/WorkerInitHandler.test.ts

```diff
@@ -150,4 +211,72 @@ describe('WorkerInitHandler', () => {
         );
         expect(channel.packageJson).to.be.empty;
     });
+
+    for (const extension of ['.js', '.mjs']) {

```

Comment:
Is there a reason `.cjs` isn't also there?

Reply:
Nope, I can add it